Firestore/Source/Local/FSTLocalStore.mm

```diff
@@ -375,9 +375,10 @@ - (nullable FSTMutationBatch *)nextMutationBatchAfterBatchID:(BatchId)batchID {
 }
 
 - (nullable FSTMaybeDocument *)readDocument:(const DocumentKey &)key {
-  return self.persistence.run("ReadDocument", [&]() -> FSTMaybeDocument *_Nullable {
-    return [self.localDocuments documentForKey:key];
-  });
+  return self.persistence.run(
```

Comment:
Seems unrelated? (No objection to keeping it though)

Reply:
Reverted.

I get that all the time and sometimes forget to undo before uploading. There's some sort of difference of opinion between my local `style.sh` and what is run by the linter on Travis. I don't know if it's because of different `clang-format` versions, or if my runs of `style.sh` end up formatting files untouched by the PR which the linter on Travis doesn't like.

Reply:
By default, ./scripts/style.sh runs across the entire repo. In addition to being overly aggressive, it's also slow as a result. Instead, run `./scripts/style.sh origin/master` (assuming you want to merge to master). This will only run it on the files that changed between HEAD and master, which is usually what you want.